src/Core/Models/Api/Response/CipherResponseModel.cs

```diff
@@ -89,11 +89,13 @@ public CipherResponseModel(CipherDetails cipher, GlobalSettings globalSettings,
             FolderId = cipher.FolderId?.ToString();
             Favorite = cipher.Favorite;
             Edit = cipher.Edit;
+            ViewPassword = cipher.ViewPassword;
         }
 
         public string FolderId { get; set; }
         public bool Favorite { get; set; }
         public bool Edit { get; set; }
+        public bool ViewPassword { get; set; }
```

Comment:
For consistency, wonder if we shouldn't make this HidePassword and use the similar construct from the collection/group directives. @kspearrin ?

Reply:
We use a `Edit` flag for `ReadOnly`, hence my attempt to follow the same convention.

Reply:
That makes sense. Thanks